docs/sources/articles/networking.md

```diff
@@ -269,7 +269,8 @@ Whether a container can talk to the world is governed by two factors.
     containers if this parameter is `1`.  Usually you will simply leave
     the Docker server at its default setting `--ip-forward=true` and
     Docker will go set `ip_forward` to `1` for you when the server
-    starts up. To check the setting or turn it on manually:
+    starts up. If you set `--ip-forward=false`, Docker will leave
```

Comment:
If you set `--ip-forward=false` and your system's kernel has it enabled, the  `--ip-forward=false` option has no effect. To check the setting on your kernel or turn it on manually: